dotCMS/src/main/java/com/dotmarketing/portlets/structure/action/EditFieldAction.java

```diff
@@ -128,39 +125,23 @@ public void processAction(ActionMapping mapping, ActionForm form, PortletConfig
 
                 Field field = (Field) req.getAttribute(WebKeys.Field.FIELD);
                 if (InodeUtils.isSet(field.getInode())) {
-                    if (field.isFixed()
-                            || (field.getFieldType().equals(Field.FieldType.LINE_DIVIDER.toString())
-                                    || field.getFieldType().equals(Field.FieldType.TAB_DIVIDER.toString())
-                                    || field.getFieldType().equals(Field.FieldType.CATEGORIES_TAB.toString())
-                                    || field.getFieldType().equals(Field.FieldType.PERMISSIONS_TAB.toString())
-                                    || field.getFieldType().equals(Field.FieldType.RELATIONSHIPS_TAB.toString())
-                                    || field.getFieldContentlet().equals(FieldAPI.ELEMENT_CONSTANT) || field
-                                    .getFieldType().equals(Field.FieldType.HIDDEN.toString()))) {
+                    if (field.isFixed()) {
+                        
                         FieldForm fieldForm = (FieldForm) form;
                         field.setFieldName(fieldForm.getFieldName());
-
-                        // This is what you can change on a fixed field
-                        if (field.isFixed()) {
-                            field.setHint(fieldForm.getHint());
-                            field.setDefaultValue(fieldForm.getDefaultValue());
-                            field.setSearchable(fieldForm.isSearchable());
-                            field.setListed(fieldForm.isListed());
-                        }
+                        field.setHint(fieldForm.getHint());
+                        field.setDefaultValue(fieldForm.getDefaultValue());
+                        field.setSearchable(fieldForm.isSearchable());
+                        field.setListed(fieldForm.isListed());
 
                         Structure contentType = CacheLocator.getContentTypeCache().getStructureByInode(field.getStructureInode());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "contentType". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "contentType" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)